db/seeds.rb

```diff
@@ -23,4 +23,7 @@
 Measure.import('db/measures/S10_outeiro.csv', 2, 'S10')
 Measure.import('db/measures/S500_outeiro.csv', 2, 'S500')
 
-Plan.update_all(finished: true)
\ No newline at end of file
+Plan.update_all(finished: true)
+
+User.create(email: 'thiagobocorrea@gmail.com', password: 'thiago00')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Final newline missing.